sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/internal/aggregator/DoubleExponentialHistogramAggregatorTest.java

```diff
@@ -122,6 +135,45 @@ void testInvalidRecording() {
     assertThat(acc.getZeroCount()).isEqualTo(0);
   }
 
+  @ParameterizedTest
+  @MethodSource("provideAggregator")
+  void testRecordingsAtLimits(DoubleExponentialHistogramAggregator aggregator) {
+    AggregatorHandle<ExponentialHistogramAccumulation> aggregatorHandle = aggregator.createHandle();
+
+    aggregatorHandle.recordDouble(Double.MIN_VALUE);
+    aggregatorHandle.recordDouble(Double.MAX_VALUE);
+
+    ExponentialHistogramAccumulation acc = aggregatorHandle.accumulateThenReset(Attributes.empty());
+    List<Long> bucketCounts = Objects.requireNonNull(acc).getPositiveBuckets().getBucketCounts();
+
+    // assert buckets == [1 0 0 0 ... 1]
+    assertThat(bucketCounts.get(0)).isEqualTo(1);
+    assertThat(bucketCounts.get(bucketCounts.size() - 1)).isEqualTo(1);
+    assertThat(bucketCounts.stream().filter(i -> i == 0).count())
+        .isEqualTo(bucketCounts.size() - 2);
+
+    // With 320 buckets, minimum scale is -3
+    assertThat(acc.getScale()).isEqualTo(-3);
+
+    // if scale is -3, base is 256.
+    int base = 256;
+
+    // Verify the rule holds:
+    // base ^ (offset+i) <= (values recorded to bucket i) < base ^ (offset+i+1)
+
+    // lowest bucket:
+    assertThat(Math.pow(base, acc.getPositiveBuckets().getOffset()))
+        .isLessThanOrEqualTo(Double.MIN_VALUE);
```

Comment:
Should this be isEqualTo? It shouldn't be possible for the double result of pow to be less than the MIN See below for related

Reply:
This test case relies on `Math.pow()` rounding down to zero. Again, it's not entirely accurate, but it works and indicates the lower bound of the bucket is placed correctly. The bucket bounds are never explicitly calculated in the main code so it never becomes a problem there. 

I tried using `BigDecimal` here: 
```java
BigDecimal lowerBound = new BigDecimal("256").pow(acc.getPositiveBuckets().getOffset());
assertThat(lowerBound)
    .isLessThanOrEqualTo(new BigDecimal("4.9E-324")); // double min value as BigDecimal
```

but the `BigDecimal.pow()` throws and says invalid operation. I'm open to suggestions on better approaches for this. Let me know if you have any ideas.

Reply:
How about `.isEqualTo(0)` and `isEqualTo(Infinity)` here and below? I think it's a bit clearer than less than or equal to min, greater than max. We could add single line comments like Math.pow rounds down to zero or up to Infinity

Reply:
Yeah that makes sense, I agree that's clearer. I've added comments and used `.isEqualTo(0)` and `isEqualTo(Infinity)`.